posthog/hogql_queries/insights/test/__snapshots__/test_lifecycle_query_runner.ambr

```diff
@@ -154,8 +154,8 @@
                0 AS counts,
                sec.status
         FROM
-          (SELECT minus(dateTrunc('day', assumeNotNull(parseDateTime64BestEffortOrNull('2020-01-19 23:59:59', 6, 'US/Pacific'))), toIntervalDay(numbers.number)) AS start_of_period
-           FROM numbers(dateDiff('day', dateTrunc('day', assumeNotNull(parseDateTime64BestEffortOrNull('2020-01-12 00:00:00', 6, 'US/Pacific'))), dateTrunc('day', plus(assumeNotNull(parseDateTime64BestEffortOrNull('2020-01-19 23:59:59', 6, 'US/Pacific')), toIntervalDay(1))))) AS numbers) AS periods
+          (SELECT minus(dateTrunc('day', assumeNotNull(parseDateTime64BestEffortOrNull('2020-01-18 23:59:59', 6, 'US/Pacific'))), toIntervalDay(numbers.number)) AS start_of_period
+           FROM numbers(dateDiff('day', dateTrunc('day', assumeNotNull(parseDateTime64BestEffortOrNull('2020-01-11 16:00:00', 6, 'US/Pacific'))), dateTrunc('day', plus(assumeNotNull(parseDateTime64BestEffortOrNull('2020-01-18 23:59:59', 6, 'US/Pacific')), toIntervalDay(1))))) AS numbers) AS periods
```

Comment:
This `16:00` still feels wrong.

Reply:
I guess that's the 8 hours of difference to midnight? Hence also the `dateTrunc` around it.
I didn't want to touch any calculations here.. this is just what the results are.

Reply:
I'm not sure we are on the same page. This is present in master. If I send this query for example

```
{
	"client_query_id": "4492fc0c-c952-41f5-a679-43b5db7f9a8d",
	"query": {
		"dateRange": {
			"date_from": "2023-11-01T00:00:00Z",
			"date_to": "2023-12-01T00:00:00Z"
		},
		"interval": "day",
		"kind": "LifecycleQuery",
		"lifecycleFilter": {},
		"series": [
			{
				"event": "$pageview",
				"kind": "EventsNode",
				"math": "total",
				"name": "$pageview"
			}
		]
	},
	"refresh": false
}
```

it makes this SQL, see the 17:00:00

<img width="951" alt="image" src="https://github.com/PostHog/posthog/assets/59713/35db0b02-2239-49eb-942d-4ddbfdf99656">

Reply:
Send while on US/Pacific in team settings I mean.

Reply:
How can that be, if `master` is the red in these snapshot files? 🤔 

I ran this locally, and first set my team to US/Pacific.

![2023-12-06 10 18 58](https://github.com/PostHog/posthog/assets/53387/77c568bd-f31f-4974-80e3-75f054ed691d)

It's the same in the generated ClickHouse SQL

Reply:
Perhaps...

```
                    "date_from": "2020-01-12T00:00:00Z",
                    "date_to": "2020-01-19T00:00:00Z",
```

... the "Z" in those dates is the culprit?

Reply:
The query in the interface does have just this:
<img width="224" alt="image" src="https://github.com/PostHog/posthog/assets/53387/25c6d7cf-e23b-43b2-9188-59086f277781">

This does make sense though if we're explicitly searching from hour to hour, however then the "date_to" part gets oddly truncated. 

To make our lives easier, we try to just ignore timezones in the frontend. It's possible this test is just sending something that you won't find in real world conditions, and needs the "Z" removed.